mattermost-jenkins/charts/jenkins/.helmignore

```diff
@@ -1,21 +0,0 @@
-# Patterns to ignore when building packages.
```

Comment:
Can you please not delete the jenkins chart? There is actually a small modification to it. Only used for the private build though. I hope to remove the need for it soon.